libraries/chain/include/eosio/chain/transaction_context.hpp

```diff
@@ -75,6 +75,7 @@ namespace eosio { namespace chain {
          fc::microseconds              delay;
          bool                          is_input           = false;
          bool                          apply_context_free = true;
+         bool                          cannot_subjectively_fail = false;
```

Comment:
The double negative name is probably more confusing than an alternative like `can_subjectively_fail = true/false`.  I realize that will invert all the usages of it but there were enough "Not cannot" checks that I got concerned